src/components/comments.jsx

```diff
@@ -1,20 +1,57 @@
-import { getCommentsByReview } from "../api";
+import { getCommentsByReview, postComment } from "../api";
 import "./review.css";
 import { useParams } from "react-router-dom";
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 
-function Comments({ comments, setComments, loading, setLoading }) {
+function Comments({ comments, setComments }) {
   const { review_id } = useParams();
+  const [newComment, setNewComment] = useState("");
+  const [loadingComments, setLoadingComments] = useState(true);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    postComment(newComment, review_id).then(() => {
```

Comment:
What happens if post comment fails? How can we let a user know that something went wrong? Also, this current allows a user on a slow connection to make multiple post requests before the initial one has succeeded. You should prevent this with a condition in state.

Reply:
I have made it so the textbox is readonly during the load so they cannot select it and the button is disabled. When comment is posted it goes back to normal. I am not sure how to process error hnadling yet but I might wait till the error ticket if that is okay so I can go through the whole application and deal with errors as one ticket?